UnitTests/Linq/BehaviorFixture.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace Moq.Tests.Linq
+{
+    using Xunit;
+
+    public class BehaviorFixture
+    {
+        [Fact]
+        public void NoQuery_Default()
+        {
+            var target = Mock.Of<IFoo>();
+            Assert.Equal(MockBehavior.Default, Mock.Get(target).Behavior);
+        }
+
+        [Fact]
+        public void WithQuery_Default()
+        {
+            var target = Mock.Of<IFoo>(x => x.BoolProperty1);
+            Assert.Equal(MockBehavior.Default, Mock.Get(target).Behavior);
+        }
+
+        [Fact]
+        public void NoQuery_Strict()
+        {
+            var target = Mock.Of<IFoo>(MockBehavior.Strict);
+            Assert.Equal(MockBehavior.Strict, Mock.Get(target).Behavior);
+        }
+
+        [Fact]
+        public void NoQuery_Loose()
+        {
+            var target = Mock.Of<IFoo>(MockBehavior.Loose);
+            Assert.Equal(MockBehavior.Loose, Mock.Get(target).Behavior);
+        }
+
+        [Fact]
+        public void WithQuery_Strict()
+        {
+            var target = Mock.Of<IFoo>(x => x.BoolProperty1, MockBehavior.Strict);
```

Comment:
Test should assign a value in the expression:

x => x.BoolProperty1 == true

And verify that it is indeed true.